Makefile

```diff
@@ -0,0 +1,61 @@
+PREFIX = /opt/obsrvbl-ossec
+TARGET = local
+VERSION = 2.8.2
+BUILD_DIR = ossec-hids-${TARGET}-${VERSION}
+TARGET_ROOT = ${BUILD_DIR}/target_root
+
+build:
+	cp -r ossec-hids/src/ ${BUILD_DIR}/
+	cp -r ossec-hids/etc/ ${BUILD_DIR}/
+	cp -r ossec-hids/contrib/ ${BUILD_DIR}/
+	cp -r ossec-hids/active-response/ ${BUILD_DIR}/
+	make -C ${BUILD_DIR} all
+	make -C ${BUILD_DIR} install
+
+deb:
+	mkdir -p packages/
+	fpm \
+		-s dir \
+		-t deb \
+		-n obsrvbl-ossec \
+		-v ${VERSION} \
+		-p packages/${BUILD_DIR}.deb \
+		--force \
+		--category 'admin' \
+		--license 'GNU General Public License (version 2)' \
+		--conflicts 'ossec-hids-server' \
+		--conflicts 'ossec-hids-agent' \
+		--deb-recommends 'ona-service' \
+		--deb-compression bzip2 \
+		--after-install ${BUILD_DIR}/scripts/ossec-hids-local.postinst \
+		--after-remove ${BUILD_DIR}/scripts/ossec-hids-local.postrm \
+		--url 'https://github.com/obsrvbl/obsrvbl-ossec/' \
+		--description 'OSSEC package for use with Observable Networks monitoring' \
+		-m "Observable Networks, Inc. <engineering@observable.net>" \
+		-a 'amd64' \
+		--deb-no-default-config-files \
```

Comment:
This is a workaround for jordansissel/fpm#975 , which is fixed in its master branch but not released yet. Once released it can be replaced with the proper configuration files.